joblib/test/test_hashing.py

```diff
@@ -259,6 +259,14 @@ def test_hash_object_dtype():
     nose.tools.assert_equal(hash(a),
                             hash(b))
 
+    from ..parallel import Parallel, delayed
+
+    args = {'np1': np.arange(10), 'np2': np.arange(20)}
+    codes = Parallel(n_jobs=3)(delayed(hash)(args) for _ in range(10))
+
+    nose.tools.assert_equal(len(set(codes)), 1)
+    nose.tools.assert_equal(codes[0], hash(args))
```

Comment:
> The test does fail in my machine ( windows 8 & fedora 20, Python 2 & 3). I don't know why it succeed in your environment .

My bad for some reason I assumed len(set(codes)) would be different than 1. This is the second assertion that actually fails, i.e. all the elements of `codes` are the same but the hashes computed by Parallel are different than the one computed directly.